Stratis.Bitcoin/Features/RPC/Models/PeerNodeModel.cs

```diff
@@ -0,0 +1,132 @@
+﻿using Newtonsoft.Json;
+
+namespace Stratis.Bitcoin.Features.RPC.Models
+{
+    /// <summary>
+    /// Data structure for RPC peer info nodes.
+    /// </summary>
+    public class PeerNodeModel
```

Comment:
I can see that a few properties are missing, like relaytxes or minping (there may be others). Can you please add them?